packages/terra-table/src/subcomponents/Cell.jsx

```diff
@@ -106,6 +106,12 @@ const propTypes = {
    * String that specifies the min-height of the cell. Any valid CSS value is accepted.
    */
   height: PropTypes.string,
+
+  /**
+   * @private
+   * True if Table is invoked from WorkListDataGrid component
+   */
+  fromWorkListDataGrid: PropTypes.bool,
```

Comment:
The Cell component should not have knowledge about the WorklistDataGrid.  We should not have a property by this name.

Reply:
Updated. 
Commit: 5cbf782d540fe32fe513e317693cf9092bd902eb
Thanks